packages/tuono/src/build/index.ts

```diff
@@ -1,6 +1,7 @@
-import type { InlineConfig } from 'vite'
+import type { InlineConfig, Plugin } from 'vite'
 import { build, createServer, mergeConfig } from 'vite'
 import react from '@vitejs/plugin-react-swc'
+import inject from '@rollup/plugin-inject'
 import ViteFsRouter from 'tuono-fs-router-vite-plugin'
 import { LazyLoadingPlugin } from 'tuono-lazy-fn-vite-plugin'
```

Comment:
Unrelated, but just noticed that `tuono-lazy-fn-vite-plugin` has a named export, 
while `tuono-fs-router-vite-plugin` has a default export. I think we should use named export in both.

I'll do it in a separate if you agree

Reply:
Agree. Thank you

Reply:
#234